app/controllers/deploys_controller.rb

```diff
@@ -1,18 +1,13 @@
 class DeploysController < ApplicationController
-  include CurrentProject
   include ProjectLevelAuthorization
 
-
-  before_action except: [:active, :active_count, :recent, :changeset] do
-    find_project(params[:project_id])
-  end
-
```

Comment:
How about we keep this `before_action`, but have it call `@project = current_project`, rather than having that line in all the actions below?  That's the point of the before_actions; to extract repeated code.